src/overlays/gamestates/ovl_file_choose/z_file_nameset_PAL.c

```diff
@@ -941,6 +943,9 @@ void FileSelect_DrawOptionsImpl(GameState* thisx) {
             gDPSetEnvColor(POLY_OPA_DISP++, 0, 0, 0, 255);
         }
 
+        //! @bug Mistakenly using sOptionsMenuHeaders instead of sOptionsMenuSettings for the height.
+        //! This is also an OOB read that happens to access the height of the first two elements in
+        //! sOptionsMenuSettings, and since all heights are 16, it works out anyway.
```

Comment:
This should probably become an AVOID_UB (at least eventually) since it relies on the variables being in the same order in memory, which isn't guaranteed

Reply:
I agree on the surface but

1) not sure if the `#if avoid_ub` guards should be around the whole command (lengthy copypaste) or only the argument (weird)
2) it feels weird to offer a fix for this under avoid_ub but not for the other similar occurrence above (which isn't UB because not OOB, just a copypaste mistake)
3) I'm not even sure what the `avoid_ub` behavior here should be (the semantics for `avoid_ub` is write alternative non-ub-invoking code that behaves the same as the ido-compiled code, not to fix things)

and I'm not a huge fan of `AVOID_UB` anyway so would rather let someone else deal with it if at all

fwiw this causes a warning when compiling with gcc (that's how I found this) so it's not entirely hidden

Reply:
I understand if you don't want to add it yourself, but I'll answer to your points anyway so we can figure out what we'll do in the future:

Since it's always 16 with IDO (as long as memory manipulation isn't involved at least), I'd say the `AVOID_UB` version would just use the right variable so the behaviour is equivalent and correct.
Also, if we feel it would be weird otherwise, I think it's okay to have a fix for the other one under `AVOID_UB`. Imo the goal of `AVOID_UB` should be to cover at least all cases of UB, but it's fine if it makes small improvements at the same time, as long as they make sense.
As for how to format it, I think it would be fine either way, tho I would probably duplicate the command